test.js

```diff
@@ -0,0 +1,255 @@
+import test from 'node:test'
+import assert from 'node:assert'
+import timers from 'node:timers/promises'
+import {
+  StuckTransactionsCanceller,
+  cancelTx,
+  getRecentSendMessage
+} from './index.js'
+import createDebug from 'debug'
+
+const debug = createDebug('test')
+
+test('StuckTransactionsCanceller', async t => {
+  await t.test('#addPending()', async () => {
+    const tx = {
+      hash: 'hash',
+      maxPriorityFeePerGas: 10n,
+      nonce: 20,
+      from: '0x0'
+    }
+    const storage = new Map()
+    const stuckTransactionsCanceller = new StuckTransactionsCanceller({
+      store: {
+        add ({ hash, timestamp, from, maxPriorityFeePerGas, nonce }) {
+          assert(!storage.has(hash))
+          assert.strictEqual(typeof hash, 'string')
+          assert.strictEqual(typeof timestamp, 'string')
+          assert.strictEqual(typeof from, 'string')
+          assert.strictEqual(typeof maxPriorityFeePerGas, 'bigint')
+          assert.strictEqual(typeof nonce, 'number')
+          storage.set(hash, {
+            hash,
+            timestamp,
+            from,
+            maxPriorityFeePerGas,
+            nonce
+          })
+        },
+        list () {
+          throw new Error('Should not be called')
+        },
+        remove (hash) {
+          throw new Error('Should not be called')
+        }
+      },
+      log: str => {
+        // TODO: Test logs
+        debug(str)
+      },
```

Comment:
Nitpick, feel free to ignore.

```suggestion
      log: str => debug(str)
```

(This also applies to all tests below.)

Reply:
See comment above